kitchen.windows.yml

```
@@ -0,0 +1,31 @@
+---
+driver:
+  name: proxy
+  host: localhost
+  reset_command: "exit 0"
+  port: 5985
+  username: kitchen
+  password: Password1
+
+provisioner:
+  salt_bootstrap_url: D:/a/salt-bootstrap/salt-bootstrap/bootstrap-salt.ps1
```

```suggestion
  salt_bootstrap_url: bootstrap-salt.ps1
```

^ I tried the obvious first but doesn't install the minion.  Probably a path issue.  Just giving the explicit path instead did the trick.